src/main/java/com/project/jfb/config/ThisIsMyFirstConditionalBean.java

```diff
@@ -0,0 +1,19 @@
+package com.project.jfb.config;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan
+public class ThisIsMyFirstConditionalBean {
+
+    @Bean
+    @ConditionalOnProperty(name = "isConditionalBeanAvailable", havingValue = "true")
+    public String useMyConditionalBean() {
+
+        return "Conditional bean is available";
+    }
```

Comment:
when you create bean, then you need not just create a method with some condition, you also need to return this been, not a `String`:
```java
    @Bean
    @ConditionalOnProperty(name = "app.allowConditionalBean", havingValue = "true")
    public ThisIsMyFirstConditionalBean thisIsMyFirstConditionalBean() {
        return new ThisIsMyFirstConditionalBean();
    }
```

Reply:
Fixed